kasa/protocol.py

```diff
@@ -133,21 +134,21 @@ async def _query(self, request: str, retry_count: int, timeout: int) -> Dict:
             except ConnectionRefusedError as ex:
                 await self.close()
                 raise SmartDeviceException(
-                    f"Unable to connect to the device: {self.host}: {ex}"
+                    f"Unable to connect to the device: {self.host}:{self.port} {ex}"
```

Comment:
```suggestion
                    f"Unable to connect to the device: {self.host}:{self.port}: {ex}"
```
Looks like the separator between the host and the exception is missing at least from exceptions.

Reply:
Updated